website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,53 @@
+---
+description: |
+  You can configure Packer to publish metadata for all completed builds to an active HCP Packer registry. 
+  Legacy JSON templates can connect to the registry using environment variables. HCL2 templates can connect using an hcp_packer_registry block. 
+page_title: Configuring HCP Packer
+---
+
+# Configuring HCP Packer
+
+[HCP Packer](https://cloud.hashicorp.com/docs/packer) stores metadata about your Packer images so that you can track updates, 
+use the most up-to-date base images, and deploy the most up-to-date downstream images. 
+
+HCP Packer can be used with both JSON and HCL2 templates. If you are using JSON templates we recommend getting started with 
+the [HCP Packer environment variables](#hcp-packer-environment-variables) and then, when ready, migrate from JSON to HCL2 templates. 
+
+~> Refer to the [HCP Packer Template Configuration](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration) documentation for details 
+on using HCL2 templates to push image metadata to an active HCP Packer registry.
+
+## HCP Packer Environment Variables
+
+The following environment variables allow you to configure Packer to push image metadata to an active registry without changing your template. 
+
+Environment variables are the only way to configure HCP Packer from a JSON template. 
+
+You must set following environment variables to enable Packer to push metadata to a registry.
+
+- `HCP_CLIENT_ID`         - The HCP client ID of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_CLIENT_SECRET`    - The HCP client secret of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_PACKER_BUCKET_NAME` -  The name of the Image bucket where Packer will store information about each artifact from a build within the configured template. 
+
+
+You can set these additional environment variables to control how metadata is pushed to the registry.
+
+- `HCP_PACKER_BUILD_FINGERPRINT` - A unique identifier assigned to a build run to determine if metadata for a template build on the registry is complete. By default, the HEAD Git SHA 
+for the template file is used to create a fingerprint. If the template is not version controlled, this environment variable must be set to a unique value before executing a build on the template. 
+
+- `HCP_PACKER_REGISTRY`    -  Allows users to toggle HCP Packer publishing off. Allowed values are [0|OFF].
+
+## HCP Packer Registry Block
+
+For HCL2 templates, the recommended approach for configuring HCP Packer is within the template via the `hcp_packer_registry` block,
+as it lets you customize the metadata Packer sent to HCP Packer registry. 
```

Comment:
```suggestion
The only metadata that Packer can infer from a template with the basic configuration are the build name and build fingerprint. For HCL2 templates, we recommend adding the `hcp_packer_registry` block to your template so that you can customize the metadata that Packer sends to the registry. 

The `hcp_packer_registry` block is only available for HCL2 Packer templates. There is no [ `PACKER_CONFIG`](docs/configure#packer-s-config-file) equivalent for JSON.
```

Reply:
Should we maybe add "environment variables" to the sentence to be more explicit on which basic configuration we are referring to? 

```
The only metadata that Packer can infer from a template with the basic environment variables configuration are the build name and build fingerprint.
```